cmd/corectl/main.go

```diff
@@ -65,6 +65,9 @@ var commands = map[string]*command{
 	"init":                 {initCluster},
 	"evict":                {evictNode},
 	"allow-address":        {allowRaftMember},
+	"get":                  {get},
```

Comment:
Agreed, this looks just fine. To clarify, my concern was about the analogous SDK/RPC call. But it looks like at that level, there is an endpoint (`/config`) that simply returns the entire config object, and in that object, all keys map to slices of strings. With that level of generality, my concern is moot.